Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.1
```

Comment:
This needs to be 5.2

Reply:
I'm using Xcode 11.3 so far, with a Swift 5.2 development snapshot. Those snapshots still see their tools version as 5.1 for some reason. Will revert it before merge, of course.